### DISCUSSION TITLE
I am stuck :(

### DISCUSSION THREAD

**Discussion by gvhomeassistant** [Dilbert66/esphome-vistaECP#97] [Q&A]
Hi, 
I tried to follow all the steps, now everything is connected, but I an totally unable to install the virtual keyboard card, 

It stays like this forever 
![image](https://user-images.githubusercontent.com/121770746/222968298-b27840ef-6027-4fb6-974a-098cfe4c05de.png)

Also, I have absolutely now idea how I can test if my ESP32 is really talking to the Vista20P panel or if all my setup is wrong ...

![image](https://user-images.githubusercontent.com/121770746/222968379-7b894dac-b0e1-4c47-8f5d-781365ac3ba6.png)

**Comment by Dilbert66** [comment]
You should see traffic logs showing cmd packets.
